include/mgba/internal/gba/cart/vfame.h

```diff
@@ -16,7 +16,8 @@ CXX_GUARD_START
 enum GBAVFameCartType {
 	VFAME_NO = 0,
 	VFAME_STANDARD = 1,
-	VFAME_GEORGE = 2
+	VFAME_ALTERNATE = 2,
+	VFAME_GEORGE = 3
```

Comment:
Should probably keep these in the original order and add ALTERNATE as 3. It doesn't really matter TOO much, but is there a strong reason not to?

Also ending the terminal entry with a comma helps reduce code churn when new entries are added.